server/main.lua

```diff
@@ -5,23 +5,29 @@ TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 RegisterCommand("cam", function(source, args, raw)          -- Camera Man and Boss Access only
     local src = source                                      -- Added 'xPlayer.job.name == 'Journalist' to prevent other jobs from accessing said props.'
     local xPlayer = ESX.GetPlayerFromId(src)
-    if xPlayer ~= nil and xPlayer.job.grade == 1 or xPlayer.job.grade == 4 and xPlayer.job.name == 'Journalist' then
-        TriggerClientEvent("Cam:ToggleCam", src)
+    if xPlayer ~= nil and xPlayer.job.name == 'Journalist' then
```

Comment:
Job name is all lowercase in ESX convention, this will return false even if the player is a Journalist.